File: app/src/main/java/com/modori/kwonkiseokee/AUto/utilities/NetworkTool.java
package com.modori.kwonkiseokee.AUto.utilities;

import android.content.Context;
import android.net.ConnectivityManager;
import android.net.NetworkInfo;

public class NetworkTool {
    public static int getNetWorkType(Context context){
        ConnectivityManager manager = (ConnectivityManager)context.getSystemService(Context.CONNECTIVITY_SERVICE);
        NetworkInfo activeNetworkInfo = manager.getActiveNetworkInfo();

        if(activeNetworkInfo != null && activeNetworkInfo.isConnected()){
            if(activeNetworkInfo.getType() == ConnectivityManager.TYPE_WIMAX || activeNetworkInfo.getType() == ConnectivityManager.TYPE_WIFI){
                return 1;
            }else{
                return 2;
            }
        }else{
            return 0;
        }


    }
}

File: app/src/main/java/com/modori/kwonkiseokee/AUto/utilities/NetWorkTool.kt
package com.modori.kwonkiseokee.AUto.utilities

import android.content.Context
import android.net.ConnectivityManager

object NetWorkTool {
    fun getNetWorkType(context: Context): Int {
        val manager = context.getSystemService(Context.CONNECTIVITY_SERVICE) as ConnectivityManager
        val activeNetworkInfo = manager.activeNetworkInfo
        return if (activeNetworkInfo != null && activeNetworkInfo.isConnected) {
            if (activeNetworkInfo.type == ConnectivityManager.TYPE_WIMAX || activeNetworkInfo.type == ConnectivityManager.TYPE_WIFI) {
                1
            } else {
                2
            }
        } else {
            0
        }
    }
}

